gym_ignition/robots/base/gazebo_robot.py

```diff
@@ -256,7 +256,15 @@ def joint_position_limits(self, joint_name: str) -> Tuple[float, float]:
         return float(limit.min), float(limit.max)
 
     def joint_force_limit(self, joint_name: str) -> float:
-        raise NotImplementedError
+        limit = self.gympp_robot.jointEffortLimit(joint_name)
+        return float(limit)
+
+    def set_joint_force_limit(self, joint_name: str, limit: float) -> bool:
+        if limit <= 0:
```

Comment:
0 seems to be a perfectly valid limit, why it has been not accepted?

Reply:
If you want to set a force limit equal to 0, you could disable the joint PID and obtain the same behaviour. Though, I think that giving the possibility to disable the PID without changing the control mode or doing other operation could be useful. Thanks!

Reply:
Fixed in https://github.com/robotology/gym-ignition/pull/152/commits/9cbb34173d00042f8f7f9f230c077ff6a9848154